### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by sbellem**
Example of the PyPI upload on the test server is at https://testpypi.python.org/pypi?name=ipld&version=0.0.1&:action=display

and on the live server at https://pypi.python.org/pypi/ipld/0.0.1

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,14 +37,24 @@
...
     author="Tim Daubenschuetz",
     author_email="tim.daubenschuetz@gmail.com",
     description="An IPLD implementation in Python",
     license="Apache-2.0",
     keywords="ipld python ipfs bigchaindb",
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'Natural Language :: English',
+        'License :: OSI Approved :: Apache Software License',
```
Comment (TimDaub) [inline setup.py]: https://github.com/bigchaindb/py-ipld/blob/master/LICENSE.md

License is MIT

Comment (sbellem) [inline setup.py]: :+1:  :smile:  https://github.com/bigchaindb/py-ipld/pull/9#discussion-diff-67656756R46

**Comment by sbellem** [comment]
We can add a `long_description` field to the `setup.py` which will display more information about the package on PyPI. See example at https://testpypi.python.org/pypi?name=ipld&version=0.0.1&:action=display

It is easier to process restructedtext than markdown for PyPI so for this reason many python packages use rst for their README, e.g.: [requests](https://github.com/kennethreitz/requests).

**Comment by TimDaub** [comment]
LGTM :+1:

**Pull request merged**